CHANGES.md

```diff
@@ -1,5 +1,12 @@
 # Changes
 
+## [0.106.0](https://github.com/googleapis/google-cloud-go/compare/v0.105.0...v0.106.0) (2022-11-09)
+
+
+### Features
+
+* **debugger:** rewrite signatures in terms of new location ([3c4b2b3](https://github.com/googleapis/google-cloud-go/commit/3c4b2b34565795537aac1661e6af2442437e34ad))
```

Comment:
Should this instead be?

`chore: bulk rewrite signatures for GA clients`

Reply:
I don't understand why ` **debugger:** rewrite signatures in terms of new location` is the sole entry for this release.

Reply:
Because it is the only API that is not in its own module.